webapp/maps_heatmaps/src/clickInfoWindow.ts

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { map } from "./index";
+import { getQueriedCollection } from "./queryDB";
+import * as firebase from "firebase";
+import { GeoFirestoreTypes } from "geofirestore";
+
+//TODO: run mocha tests on the browser.
+let markers: Array<google.maps.Marker> = [];
+
+/* querys for 20 random docs in the database in order to place markers on them. */
+//TODO: make this function more random.
+//TODO: use this function to show images on the side pannel-so they will correlate (relocate to a different file)
+async function setFirstTwentyMarkers(
+  center: firebase.firestore.GeoPoint,
+  radius: number,
+  labels: string[],
+  year?: number,
+  month?: number
```

Comment:
What if we add a day or time here?
Can you please create an interface `datetime` which is optional and has optional fields year, month and day?
see: https://www.typescriptlang.org/docs/handbook/interfaces.html

Reply:
Great idea I created an interface and updated queryDB.ts and queryDB.spec.ts as well.

Reply:
Very nice, thank you :+1: